lib/gradle-wrapper.ts

```diff
@@ -1,12 +1,13 @@
 import 'source-map-support/register';
 import { execute } from './sub-process';
 import * as path from 'path';
+import {EOL, platform} from 'os';
```

Comment:
Just make sure to have consistent formatting here, you should have spaces before and after the curly braces, `{ EOL, platform }`